### ISSUE TITLE
exists is not defined in `ValidationParamSchema`

### ISSUE THREAD

**Issue by ToonvanStrijp** [ctavan/express-validator#528]
I'm using TypeScript, I'm using the `checkSchema` but whenever I try to use the 'exists' it seems it's not defined in the `ValidationParamSchema` 

```
interface ValidationParamSchema {
  in: Location | Location[],
  errorMessage?: any

  custom?: ValidatorSchemaOptions<CustomValidator>;
  optional?: boolean | ValidatorOptions.OptionalOptions;

  equals?: ValidatorSchemaOptions;
  contains?: ValidatorSchemaOptions;
  isAfter?: ValidatorSchemaOptions;
  isAlpha?: ValidatorSchemaOptions;
  isAlphanumeric?: ValidatorSchemaOptions;
  isAscii?: ValidatorSchemaOptions;
  isBase64?: ValidatorSchemaOptions;
  isBefore?: ValidatorSchemaOptions;
  isBoolean?: ValidatorSchemaOptions;
  isByteLength?: ValidatorSchemaOptions;
  isCreditCard?: ValidatorSchemaOptions;
  isCurrency?: ValidatorSchemaOptions;
  isDataURI?: ValidatorSchemaOptions;
  isDate?: ValidatorSchemaOptions;
  isDecimal?: ValidatorSchemaOptions;
  isDivisibleBy?: ValidatorSchemaOptions;
  isEmail?: ValidatorSchemaOptions;
  isEmpty?: ValidatorSchemaOptions;
  isFloat?: ValidatorSchemaOptions;
  isFQDN?: ValidatorSchemaOptions;
  isFullWidth?: ValidatorSchemaOptions;
  isHalfWidth?: ValidatorSchemaOptions;
  isHash?: ValidatorSchemaOptions;
  isHexadecimal?: ValidatorSchemaOptions;
  isHexColor?: ValidatorSchemaOptions;
  isIn?: ValidatorSchemaOptions;
  isInt?: ValidatorSchemaOptions;
  isIP?: ValidatorSchemaOptions;
  isISIN?: ValidatorSchemaOptions;
  isISO31661Alpha2?: ValidatorSchemaOptions;
  isISO8601?: ValidatorSchemaOptions;
  isISRC?: ValidatorSchemaOptions;
  isISBN?: ValidatorSchemaOptions;
  isISSN?: ValidatorSchemaOptions;
  isJSON?: ValidatorSchemaOptions;
  isNumeric?: ValidatorSchemaOptions;
  isLatLong?: ValidatorSchemaOptions;
  isLength?: ValidatorSchemaOptions;
  isLowercase?: ValidatorSchemaOptions;
  isMACAddress?: ValidatorSchemaOptions;
  isMD5?: ValidatorSchemaOptions;
  isMimeType?: ValidatorSchemaOptions;
  isMobilePhone?: ValidatorSchemaOptions;
  isMongoId?: ValidatorSchemaOptions;
  isMultibyte?: ValidatorSchemaOptions;
  isPostalCode?: ValidatorSchemaOptions;
  isPort?: ValidatorSchemaOptions;
  isSurrogatePair?: ValidatorSchemaOptions;
  isUppercase?: ValidatorSchemaOptions;
  isURL?: ValidatorSchemaOptions;
  isUUID?: ValidatorSchemaOptions;
  isVariableWidth?: ValidatorSchemaOptions;
  isWhitelisted?: ValidatorSchemaOptions;
  matches?: ValidatorSchemaOptions;
}
```
The code I'm using:

```
checkSchema({
            email: {
                in: ['body'],
                exists: {
                    errorMessage: 'Email is verplicht.'
                }
            },
            password: {
                in: ['body'],
                exists: {
                    errorMessage: 'Wachtwoord is verplicht.'
                }
            }
        })
```

Does someone know how to fix this? Or is it an issue with the types definition of this library?

**Comment by gustavohenke** [comment]
Duplicate of #525

**Comment by gustavohenke** [comment]
Fix pushed to v5.0.1.
